app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java

```diff
@@ -85,6 +86,12 @@ public void onCreate(Bundle savedInstanceState) {
 
         loginButton.setOnClickListener(view -> performLogin());
         signupButton.setOnClickListener(view -> signUp());
+
+        if(BuildConfig.FLAVOR == "beta"){
+            loginCredentials.setText("Use Commons Beta Credentials");
+        } else {
```

Comment:
Can you please move this into strings.xml, so that we can get translations. Besides instead of imperative sentence, Isn't it better an informing sentence as "You are using Commons Beta Credentials" ? By the way I am not an English expert definitely, what do you think @misaochan ?

Reply:
Will do !

Reply:
I agree that this should go into strings.xml. For the text itself, not so important since the beta flavour will never be seen by actual users, but I would personally go with "Log in to your Commons Beta account").

Reply:
will do !